.github/workflows/migration-tests.yml

```diff
@@ -461,3 +464,27 @@ jobs:
           MONGODB_ATLAS_LAST_VERSION: ${{ needs.get-provider-version.outputs.provider_version }}
           TEST_REGEX: "^TestAccMigrationcDataLake"
         run: make testacc
+  cluster_outage_simulation:
+    needs: [ change-detection, get-provider-version ]
+    if: ${{ needs.change-detection.outputs.cluster_outage_simulation == 'true' || inputs.test_group == 'cluster_outage_simulation' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
```

Comment:
let's use sha instead of tag

Reply:
I think it is 🤔 , see https://github.com/mongodb/terraform-provider-mongodbatlas/blob/d5c7ea8e7dd579b9aeb4107ce16d081cc811d550/.github/workflows/acceptance-tests-runner.yml#L228

Reply:
FYI: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1858

Reply:
uhm, you're right, but it's not consistent, not all step in all actions have it

Reply:
I changed all actions in `migration-tests.yml` to use SHA